app/assets/src/components/views/projects/ProjectsView.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Table } from "~/components/visualizations/table";
+import { merge, pick } from "lodash";
```

Comment:
@MarkAZhang I've noticed some of these regular "lodash" ones slipping in.. should we still be preferring lodash/fp? Tbh I've been thrown off by lodash/fp args sometimes because most code snippets online use regular lodash

Reply:
I think it would be good to be consistent and use `lodash/fp`. It looks like we use it in 90% of our files.

Reply:
To be precise is 32 / 15 import lines for `lodash/fp`. :)
I gave it a try and I really dislike the args opinions of `lodash/fp` and I think it is affecting the code readability for me (maybe just a personal issue). 

Would hope we can reconsider here... 
(I am normally all for consistency, and pushed for it in this case at the beginning but in this case I would prefer ditching `lodash/fp`)
We can discuss this offline...